docs/using-the-jdbc-driver/ReaderSelectionStrategies.md

```diff
@@ -0,0 +1,16 @@
+## Reader Selection Strategies
+By default, the read/write plugin randomly selects a reader instance the first time that `setReadOnly(true)` is called. To balance connections to reader instances more evenly, different selection strategies can be used. The following table describes the currently available selection strategies and any relevant configuration parameters for each strategy.
+
+To indicate which selection strategy to use, the `readerHostSelectorStrategy` configuration parameter can be set to one of the selection strategies in the table below. The following is an example of enabling the least connections strategy:
+
```

Comment:
This should be left on RW splitting plugin page

Reply:
Moved this back